java/pom.xml

```diff
@@ -26,8 +26,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.0</version>
 				<configuration>
-					<source>1.3</source>
-					<target>1.1</target>
+					<source>1.5</source>
+					<target>1.5</target>
```

Comment:
good callout here - we have some < 1.5 java meterpreter compat code listed in the archetype samples too that could be cleaned up later